compiler/rustc_apfloat/src/ieee.rs

```diff
@@ -2646,7 +2646,7 @@ mod sig {
 
         // Convert the result having "2 * precision" significant-bits back to the one
         // having "precision" significant-bits. First, move the radix point from
-        // poision "2*precision - 1" to "precision - 1". The exponent need to be
+        // position "2*precision - 1" to "precision - 1". The exponent need to be
```

Comment:
This should probably not be changed. The license situation around rustc_apfloat is unclear.

Reply:
FWIW this PR still changes apfloat

Reply:
@ralfjung: this doesn't seem to change apfloat on the most up to date set of changes?

Reply:
Oh weird, not sure what happened... I did load the page after you r+'d but it still showed an old diff I guess? Must be that 'eventual consistency' they always talk about...

Well sorry for the noise then.